packages/fluent/scss/pivotgrid/index.scss

```diff
@@ -32,5 +34,9 @@ $_kendo-module-meta: (
         @include kendo-pivotgrid--theme();
         @include kendo-pivotgrid-configurator--theme();
         @include kendo-pivotgrid-legacy--theme();
+
```

Comment:
Are these empty lines left on purpose for readability?

Reply:
Nope, I will remove them